src/main/scala/org/kframework/minikore/TextToMini.scala

```diff
@@ -38,7 +38,8 @@ class TextToMini {
   private def parseDefinition(): Definition = {
     val att = parseAttributes()
     val modules = parseModules(Seq())
-    Definition(modules, att)
+
```

Comment:
Kore data structures cannot be directly created anymore, hence, the parser used the default constructor implementation.